turk-one/app/Views-Controllers/Admin/admin-workers.html

```diff
@@ -190,11 +193,11 @@ <h6 class="bottom-buffer-25">References available</h6>
 
 				<p><span class="text-muted">explanation: </span> {{explain}}</p>
 
-				<p><span class="text-muted">time spent:</span> {{timeInMinutes}}m</p>
+				<p><span class="text-muted">time spent:</span> {{_time}}</p>
```

Comment:
I just changed it because the termination "inMinutes" became weird after our format change. By doing that, some template methods had the exactly same name as the context fields, what was confusing (and led to interpreter errors). So, in order to avoid inconsistent naming, I changed the others too.

Please let me know if there is a best way / guideline for doing so.